src/xmlrpc.php

```diff
@@ -25,6 +25,7 @@
 if ( isset( $HTTP_RAW_POST_DATA ) ) {
 	$HTTP_RAW_POST_DATA = trim( $HTTP_RAW_POST_DATA );
```

Comment:
I wonder if we should even bother to populate `$HTTP_RAW_POST_DATA` here. This variable does not appear to be used anywhere else in the code.  I tested with the variable removed, and [tests](https://travis-ci.com/github/Ayesh/wordpress-develop/builds/169335242) are still green.

Reply:
Quote from the comment I left in the ticket as refered to above:

> This is the only one where a case could be made for letting the variable remain as-is. The variable gets backfilled in the global scope for a very select entry point and is subsequently never used in WP Core, though the `class-IXR-server.php` file may have used it, but as it already had its own BC-code in place that was never an issue.
> 
> **Conclusion**: As it is never used in Core, this might be an entry point for which plugins expect the variable to exist and leaving it in place doesn't do any actual harm.

Reply:
I apologize for not taking a thorough look at the ticket. I think this is the right call, considering the unpleasant yet not impossible case of plugins using `$HTTP_RAW_POST_DATA`.

Reply:
No apology needed ;-) I always appreciate feedback.